quickstart_lists/README.md

```diff
@@ -0,0 +1,62 @@
+# Habitat seed lists
```

Comment:
Note that if we change the naming, the folder name should also reflect the new terminology.  Can wait to do that after naming is finalized.